### ISSUE TITLE
Consider adding a default entry for empty changelog release entries in monorepos

### ISSUE THREAD

**Issue by rekmarks** [MetaMask/action-create-release-pr#71]
Currently, if a monorepo package is released during a major version bump but hasn't changed, the changelog is by default empty. We should consider adding some kind of default so as to not confuse readers.

**Comment by Gudahtt** [comment]
It would be somewhat complicated to determine whether there was a real breaking change or not, since peer dependency changes might sometimes be breaking and other times not. I'm not sure we should attempt automating this - we might be better off just manually writing a note explaining whether it's breaking or not, and why.

**Comment by rekmarks** [comment]
I think we're talking past each other. I'm not trying to automate anything that isn't already automated, I'm suggesting that we add a default change entry in cases where there were no changes whatsoever. Currently, the entire release entry is empty when that happens. Motivated by your comment here: https://github.com/MetaMask/eslint-config/pull/198#discussion_r684397097

**Comment by rekmarks** [comment]
Hm, well for major version bumps, we could certainly do what you've done here: https://github.com/MetaMask/eslint-config/pull/212/files

**Comment by Gudahtt** [comment]
Not necessarily. There might have been other breaking changes aside from the peer dependency change. Also there might not be a peer dependency change (e.g. what if one of the "leaf" packages was the one that had the breaking change).
